app/upgrades/v2/upgrades.go

```diff
@@ -0,0 +1,40 @@
+package v2
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	"github.com/sge-network/sge/app/keepers"
+)
+
+// betOddsTypeRemoval removes bet odds type from current state
+func betOddsTypeRemoval(keepers *keepers.AppKeepers, ctx sdk.Context) {
+	oldBets, err := keepers.BetKeeper.GetBets(ctx)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, old := range oldBets {
+		betID, found := keepers.BetKeeper.GetBetID(ctx, old.UID)
+		if !found {
+			panic(fmt.Errorf("bet id for the uid not found %s", old.UID))
+		}
+
+		keepers.BetKeeper.SetBet(ctx, old, betID.ID)
+	}
```

Comment:
should we change old bets data? 

old.Metadata = {SelectedOddsType: old.OddType, SelectedOddsValue: old.OddsValue}
old.OddValue = convert old odd value to decimal.
that would need convertor code for odd types. 

Is this needed @scorpioborn .